sdk/python/kfp/_local_client.py

```diff
@@ -395,8 +414,9 @@ def _run_group_dag(
                     cmd = self._generate_cmd_for_docker_execution(
                         run_name, pipeline, op, stack
                     )
+
                 process = subprocess.Popen(
-                    subprocess.list2cmdline(cmd) if "sh" != cmd[0] else cmd,
```

Comment:
What if we just pass `cmd` here and never set `shell` to `True`?

Reply:
@Ark-kun 
I have checked this code, that you suggested.
And it works well in both DOCKER mode and LOCAL mode.
I think this is better idea than mine.